src/NUnitEngine/nunit.engine.tests/nunit.engine.tests.csproj

```diff
@@ -14,6 +14,9 @@
     <PackageReference Include="NUnit" Version="3.13.0" />
     <PackageReference Include="NSubstitute" Version="2.0.3" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)'=='netcoreapp2.1' OR '$(TargetFramework)'=='netcoreapp3.1'">
+    <Reference Include="System.Configuration" />
+  </ItemGroup>
```

Comment:
This change is also in the msi commit, and I presume also accidental?

Reply:
I had to add this in a few places before all the tests passed. I'll test removing them one at a time.

Alternatively, we could remove that single test from our .NET Core builds for the time being.